extensions/applicationinsights-debugplugin-js/package.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "@microsoft/applicationinsights-debugplugin-js",
+    "version": "2.5.4",
+    "description": "Microsoft Application Insights Javascript SDK apis",
+    "main": "dist/applicationinsights-debugplugin-js.js",
+    "module": "dist-esm/applicationinsights-debugplugin-js.js",
+    "types": "types/applicationinsights-debugplugin-js.d.ts",
+    "sideEffects": false,
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/microsoft/ApplicationInsights-JS/tree/master/extensions/applicationinsights-debugplugin-js"
+    },
+    "scripts": {
+        "build": "npm run build:esm && npm run build:browser",
+        "build:esm": "grunt debugplugin",
+        "build:browser": "rollup -c",
+        "test": "grunt aitests",
+        "lint": "tslint -p tsconfig.json"
+    },
+    "devDependencies": {
+        "@microsoft/applicationinsights-rollup-es3": "1.1.1",
+        "typescript": "2.5.3",
+        "grunt": "1.0.1",
+        "grunt-contrib-qunit": "2.0.0",
+        "grunt-contrib-uglify": "3.1.0",
+        "grunt-ts": "^6.0.0-beta.15",
+        "rollup-plugin-node-resolve": "^3.4.0",
+        "rollup-plugin-replace": "^2.1.0",
+        "rollup-plugin-uglify": "^6.0.0",
+        "rollup": "^0.66.0",
+        "tslint": "^5.19.0",
+        "tslint-config-prettier": "^1.18.0"
+    },
+    "dependencies": {
+        "@microsoft/dynamicproto-js": "^0.5.2",
+        "@microsoft/applicationinsights-common": "^2.5.5",
```

Comment:
These should actually be the specific version (2.5.5) so that rush correctly creates local links rather than dragging down the published versions. (i.e. if we add specific util functions to common/core that this required it wouldn't pick them up)